azure-pipelines.yml

```diff
@@ -142,10 +142,17 @@ jobs:
     steps:
       - template: '/miscellaneous/build_templates/appsettings.yml@templates'
 
-      - script: pip install -r requirements.txt
-        displayName: 'Install requirements'
-
-      - script: python -m pytest --junitxml=junit/test-results.xml ./test.py
+      - script: |
+          echo Install tools
+          pip install setuptools wheel
+          pip install unittest-xml-reporting    
+          pip install pytest
+          pip install pytest-cov
+          echo Install requirements
+          pip install -r requirements.txt
+          echo Run tests
+          python -m pytest ./test.py --junitxml=junit/test-results.xml 
+          echo Complete
```

Comment:
should this be templated somehow? this sequence of steps has come up in a few pipelines now